internal/component/database_observability/mysql/collector/query_tables_test.go

```diff
@@ -22,13 +22,14 @@ func TestQueryTables(t *testing.T) {
 
 	testcases := []struct {
 		name       string
-		rows       [][]driver.Value
+		qRows      [][]driver.Value
```

Comment:
can we expand `qRows` and `psRows` - the naming is quite vague.
It's not intuitive qRows means rows from `events_statements_summary_by_digest` table

Reply:
They stand for `query` rows and `prepared statement` rows. What (possibly short) name do you suggest?

Reply:
`eventStatementsRows` and `preparedStatementRows`?
or `esRows` and `psRows`